README.md

```diff
@@ -36,7 +36,7 @@ Your answer here.
 What is the purpose of the `<head></head>` tag in HTML?
 
 ```text
-Your answer here.
+The head tag contains the background information about the page including the title, image links, script links, etc.
```

Comment:
:+1: 

"Background information" is a little misleading since it sounds like you're talking about background images and background colors.